.github/workflows/ci.yml

```
@@ -108,13 +108,13 @@ jobs:
           !contains(matrix.platform.target, 'ios'))
         run: cargo $CMD test --verbose --target ${{ matrix.platform.target }} $OPTIONS --features serde,$FEATURES
 
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
```

The recommended way is it to use `@nightly `.

```suggestion
      - uses: dtolnay/rust-toolchain@nightly
```

---

will open a PR for this one

---

https://github.com/tauri-apps/tao/pull/957